src/deadline/nuke_submitter/assets.py

```diff
@@ -81,17 +81,41 @@ def get_scene_asset_references() -> AssetReferences:
             for filename in get_node_filenames(node):
                 asset_references.output_directories.add(dirname(filename))
 
-    # if using a custom OCIO config, add the config file and associated search directories
-    if nuke_ocio.is_custom_config_enabled():
-        ocio_config_path = nuke_ocio.get_custom_config_path()
-        ocio_config_search_paths = nuke_ocio.get_config_absolute_search_paths(ocio_config_path)
+    if nuke_ocio.is_OCIO_enabled():
+        # Determine and add the config file and associated search directories
+        ocio_config_path = get_ocio_config_path()
+        # Add the references
+        if ocio_config_path is not None:
+            if os.path.isfile(ocio_config_path):
+                asset_references.input_filenames.add(ocio_config_path)
+
+                ocio_config_search_paths = nuke_ocio.get_config_absolute_search_paths(
+                    ocio_config_path
+                )
+                for search_path in ocio_config_search_paths:
+                    asset_references.input_directories.add(search_path)
+            else:
+                nuke.alert(
+                    "OCIO config file specified(%s) is not an existing file." % ocio_config_path
+                )
```

Comment:
Should this error block the submission? What happens if it's missing?

Reply:
If its missing, it means that OCIO is enabled but the provided file is not an existing file or working file. In this case, it will not open Submitter pop up and instead will show the alert message. 
<img width="535" alt="Screenshot 2024-09-17 at 11 29 59 AM" src="https://github.com/user-attachments/assets/b923b3f4-5729-40cb-a791-b94bffc0c441">

Reply:
Actually, I take this back. It will still show the Submitter pop up after this alert. I will make changes to not show the submitter if the OCIO config file is not specified properly.